packages/world/ts/config/v2/dynamicResolution.ts

```diff
@@ -1,10 +1,8 @@
-export enum DynamicResolutionType {
-  TABLE_ID,
-  SYSTEM_ADDRESS,
-}
+import { World } from "./output";
 
 export type DynamicResolution = {
-  type: DynamicResolutionType;
+  // TODO: add systemAddress support
+  type: "tableId";
```

Comment:
this is an internal thing and changing this to strings instead of enums leads to better errors in the `resolveWithContext` function